src/gcm_request.erl

```diff
@@ -143,15 +143,16 @@ filter(V) when is_list(V) ->
     try 
         list_to_binary(V)
     catch
-        _:_ -> <<"">>
+        _:_ -> 
+            jsx:encode(V)
```

Comment:
I'm a little bit concerned here. If encoding crashes here, will it result in no PN at all?